src/Makefile.am

```diff
@@ -593,6 +593,7 @@ libbitcoin_common_a_SOURCES = \
   deploymentinfo.cpp \
   external_signer.cpp \
   init/common.cpp \
+  kernel/init/common.cpp \
```

Comment:
In commit "kernel: Decouple from full init/common.cpp" (9211d91b226ff45c1eeaa4942a7b5c79f170a78d)

Adding `src/kernel/` code to `libbitcoin_common` is a red flag here because it means wallet and gui code will now be depending on libbitcoinkernel source code. But I already wrote yesterday https://github.com/bitcoin/bitcoin/pjustull/24332#issuecomment-1039562660 that I don't think these SetGlobals/UnsetGlobals functions were moved to the right place anyway, so this is just additional confirmation.

A simple fix should be to just move `kernel/init/common.cpp` to `src/util/globals.cpp` and `kernel/util/common.h` to `src/util/globals.h` and list them in `libbitcoin_util_a_SOURCES` not `libbitcoin_util_a_SOURCES`. I hope reasoning makes sense that:

- It's ok for wallet code to depend on _util_ but not for it to depend on _kernel_
- Generally when we create new source files and add them to a `libbitcoin_whatever_SOURCES` we should try to locate those files in `src/whatever/`
- It would also be reasonable to add these SetGlobals/UnsetGlobals functions to _common_ instead of _util_. But I think functionally they make a little more sense in _util_ since they are lower level, and in general I think it is probably  nicer if libbitcoin_kernel doesn't depend on anything in _common_ at all. (_util_ seems like a better home for essential stuff, while _common_ seems like more of a home for miscellaneous stuff)